### ISSUE TITLE
Event Manager Dates

### ISSUE THREAD

**Issue by graylion-wa** [worldanvil/wa-chronos-bugtracker#64]
**World Anvil Username**:

Graylion 

**Chronicle Title**:

Ptolus Chronicle

**Chronos System**

Event Manager

**New Chronicle or Existing? (Chronicle created before March 16 2022)**

Existing

**Describe the Issue**

This event manager display dates appear to be off in year and in month names and date.

**Expected behavior**

The event dates and the events listing in the event manager have matching dates.

**Screenshots**

![Chronos_-_Event_Manager_Dates.jpg](https://user-images.githubusercontent.com/79761754/159355312-1be829dd-d67e-43a9-addf-4c76480c0dbb.jpg)

**Comment by Xuroth** [comment]
I presume that you are using the default Gregorian calendar with this Chronicle? If so, it's possible that the issue is due to a 'baked in' bug that's created when the original calendar was saved. If you go into the Calendar Editor (on the Toolbar) and save the calendar again, do these issues remain?

**Comment by graylion-wa** [comment]
It's not the Gregorian calendar, it the City of Ptolus.

See attached which would make for a great default also. No lead years, no festivals outside of months.

I saved the calendar again.

Refresh and open the event manager which still shows Gregorian months and Ptolus years.

![The-Ptolus-Year-Calendar-Page-571.jpg](https://user-images.githubusercontent.com/79761754/159866730-998c9a17-0f4f-4e44-80b1-38be7dd0895d.jpg)

**Comment by graylion-wa** [comment]
Note the event month and year from clicking the event.

These are correct

![Screenshot_20220324-034314_Chrome.jpg](https://user-images.githubusercontent.com/79761754/159867164-a94906af-f1de-495a-b197-455388445fed.jpg)



Note the event in the event manager the month is Gregorian and the year is off.

![Screenshot_20220324-034326_Chrome.jpg](https://user-images.githubusercontent.com/79761754/159867183-845b69dc-bf55-4834-a711-e1aca806d00c.jpg)

Thank you

**Comment by Xuroth** [comment]
Okay, try re-saving once more. I just pushed an update that fixes the Gregorian months/time units from overriding everything. Afterwards, do a hard refresh of the browser tab, and if the date is showing incorrectly, try to edit the event dates to recalculate them. If the dates are still off, you can try creating a duplicate of the event from scratch and see if the issue persists. Theoretically, the system just needs to recalculate the Calendar, but it may need to recalculate the actual dates (after the Calendar is corrected)

**Comment by graylion-wa** [comment]
OK this is all from mobile I will check from desktop later.

Clearned the cache and all mobile browser data, which forced a re-login.

Re-Saved the calendar.

Event manager month still show Gregorian and years off.

Open e e to and resaved, problem still shows in event manager.

Deleted 2 and created 2 new ones which have the same problems.

Will check on computer later. 

Thanks

![Screenshot_20220324-134150_Chrome.jpg](https://user-images.githubusercontent.com/79761754/159978176-0fc3eba2-159d-475c-b82b-8f2d8a68e525.jpg)

**Comment by graylion-wa** [comment]
OK update I tried the same items listed above on my desktop and had the same result listed above.

I decided to delete both he old Chronicle because I didn't have many events logged.

I created a new Chronicle.
Selected the Pathfinder/Galorian default.
Edited the month names and days.

The errors on year and date persist on the new Chronicle when viewing the events in the event manager.


![Screenshot_20220324-170154_Chrome.jpg](https://user-images.githubusercontent.com/79761754/160010998-e65a714b-5eba-4c21-bb77-f2adc01cb497.jpg)

![Screenshot_20220324-170206_Chrome.jpg](https://user-images.githubusercontent.com/79761754/160011019-17a5a52c-84b5-4c86-9d76-5d5797504390.jpg)
